src/Decorator/Core/Content/Product/SalesChannel/Search/ProductSearchRoute.php

```diff
@@ -4,12 +4,11 @@
 
 namespace Nosto\NostoIntegration\Decorator\Core\Content\Product\SalesChannel\Search;
 
+use Exception;
 use Nosto\NostoIntegration\Model\ConfigProvider;
 use Nosto\NostoIntegration\Traits\SearchResultHelper;
 use Nosto\NostoIntegration\Utils\SearchHelper;
 use Shopware\Core\Content\Product\Aggregate\ProductVisibility\ProductVisibilityDefinition;
-use Shopware\Core\Content\Product\Events\ProductSearchResultEvent;
-use Shopware\Core\Content\Product\ProductEvents;
```

Comment:
Seems like you left this dangling on #63

Reply:
We need it once again, so they are returned back